Microsoft.Azure.Cosmos/src/Encryption/CachedDekProperties.cs

```diff
@@ -0,0 +1,27 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Cosmos
+{
+    using System;
+
+    internal class CachedDekProperties
+    {
+        public string DatabaseId { get; }
+
+        public DataEncryptionKeyProperties ServerProperties { get;  }
+
+        public DateTime ServerPropertiesExpiry { get; }
```

Comment:
How about adding UTC to the name to avoid confusion if it should be local time or utc time?
```suggestion
        public DateTime ServerPropertiesExpiryUtc { get; }
```

Reply:
Sure.